website/docs/workloads/HPLinpack/HPLinpack-profiles.md

```diff
@@ -0,0 +1,66 @@
+﻿# HPLinpack Workload Profiles
```

Comment:
Let's stick with the casing conventions we have in place for the docs if you don't mind. It matters for ensuring we don't run into casing issues in the URIs. As such, the folder names should be lower-cased as should the file names. You can look at the other folders in the /docs/workloads directory for examples.  Thanks!